tests/rptest/test_suite_ec2.yml

```diff
@@ -3,10 +3,10 @@ ec2:
   included:
     - tests
     - scale_tests
+    - rp_cloud_tests
```

Comment:
I bet that this should go to 'excluded' instead. For example max_connection test will not work without cloud. At least not yet.